readme.md

```diff
@@ -71,6 +74,27 @@ Default: `false`
 
 Uppercase the first character: `foo-bar` → `FooBar`
 
+##### locale
+
+Type: `string | string[]`\
+Default: The host environment’s current locale
+
+*From `String.prototype.toLocaleUpperCase()`*: The locale parameter indicates the locale to be used to convert to upper/lower case according to any locale-specific case mappings. If multiple locales are given in an Array, the best available locale is used. The default locale is the host environment’s current locale.
+
+```js
+camelCase('lorem-ipsum', {locale: 'en-US'});
+//=> 'loremIpsum'
+
+camelCase('lorem-ipsum', {locale: 'tr-TR'});
+//=> 'loremİpsum'
+
+camelCase('lorem-ipsum', {locale: ['en-US', 'en-GB']});
+//=> 'loremIpsum'
+
+camelCase('lorem-ipsum', {locale: ['tr', 'TR', 'tr-TR']});
+//=> 'loremİpsum'
+```
```

Comment:
> The readme and index.d.ts should be fully in sync.

Reply:
readme contains 4 examples and index.d.ts contains one example due to your previous comment: https://github.com/sindresorhus/camelcase/pull/75#discussion_r456964680
Do you means smth else?

Reply:
These 4 examples should be here: https://github.com/sindresorhus/camelcase/pull/75/files#diff-b52768974e6bc0faccb7d4b75b162c99R14 (Just like the readme).

Reply:
Oh, ok. thanks. Done.